selectrum.el

```diff
@@ -80,6 +80,10 @@ parts of the input."
   'selectrum-should-sort
   "3.0")
 
+(defcustom selectrum-default-value-format " [default value: %s]"
```

Comment:
You can shorten this to `[default:...]` this way you don't need to configure it personally and I generally think the shorter form is better, too :)

Reply:
Haha okay.